### PR TITLE
refactor: move *index constants out of validation

### PR DESCRIPTION
**Pull request by fanquake**
Move `*index` default constants out of `validation.h`.

### CODE REVIEW

**Review by stickies-v — APPROVED** [review]
ACK 3521e630b - more sensible place for these constants to live

File: src/node/caches.cpp [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 #include <node/caches.h>
 
+#include <index/txindex.h>
 #include <txdb.h>
 #include <util/system.h>
 #include <validation.h>
```
[the remark this answers is not in the document]
Comment (fanquake) [reply src/node/caches.cpp]: Dropped.

**Comment by stickies-v** [comment]
re-ACK 7d14577d0f9316feef3bcb5220aa3037748615d3

**Review by aureleoules — APPROVED** [review]
ACK 7d14577d0f9316feef3bcb5220aa3037748615d3

**Pull request merged**